styles.css

```diff
@@ -173,3 +173,22 @@ textarea#title {
   margin-top: -16px;
   filter: brightness(.75);
 }
+
```

Comment:
@butlertree A couple of notes:
1. If you take a look at the comp, I think the button sizing is a little off. We might need to increase the height of the button itself, and maybe the font size just a hair bigger (also probably need to make the sidebar a little wider, but that could be done in a different ticket).
2. Should we start with the button disabled? It probably should be inactive unless there actually are faves to show, I'm thinking.

Other than those bits, this looks great! Nice job on the hover state! 🙂

Reply:
I think we should do a finale clean up together on the exact size of fonts and subtle adjustments to widths and heights. We could do this together tomorrow so that we all agree.

Reply:
@butlertree That's fine regarding the first point, for sure. I'm down with attacking tweaks at the end as a team.

I think the button starting out disabled should ideally be addressed here with the work for adding the button though. I'd still recommend getting that done before we merge this work in.